src/bootlint.js

```diff
@@ -617,6 +617,329 @@ var LocationIndex = _location.LocationIndex;
+            'col-md-push-6',
+            'col-md-push-7',
+            'col-md-push-8',
+            'col-md-push-9',
+            'col-md-push-10',
+            'col-md-push-11',
+            'col-md-push-12',
+            'col-sm-offset-0',
+            'col-sm-offset-1',
+            'col-sm-offset-2',
+            'col-sm-offset-3',
+            'col-sm-offset-4',
+            'col-sm-offset-5',
+            'col-sm-offset-6',
+            'col-sm-offset-7',
+            'col-sm-offset-8',
+            'col-sm-offset-9',
+            'col-sm-offset-10',
+            'col-sm-offset-11',
+            'col-sm-offset-12',
+            'col-sm-pull-0',
+            'col-sm-pull-1',
+            'col-sm-pull-2',
+            'col-sm-pull-3',
+            'col-sm-pull-4',
+            'col-sm-pull-5',
+            'col-sm-pull-6',
+            'col-sm-pull-7',
+            'col-sm-pull-8',
+            'col-sm-pull-9',
+            'col-sm-pull-10',
+            'col-sm-pull-11',
+            'col-sm-pull-12',
+            'col-sm-push-0',
+            'col-sm-push-1',
+            'col-sm-push-2',
+            'col-sm-push-3',
+            'col-sm-push-4',
+            'col-sm-push-5',
+            'col-sm-push-6',
+            'col-sm-push-7',
+            'col-sm-push-8',
+            'col-sm-push-9',
+            'col-sm-push-10',
+            'col-sm-push-11',
+            'col-sm-push-12',
+            'col-xs-1',
+            'col-xs-2',
+            'col-xs-3',
+            'col-xs-4',
+            'col-xs-5',
+            'col-xs-6',
+            'col-xs-7',
+            'col-xs-8',
+            'col-xs-9',
+            'col-xs-10',
+            'col-xs-11',
+            'col-xs-12',
+            'col-xs-offset-0',
+            'col-xs-offset-1',
+            'col-xs-offset-2',
+            'col-xs-offset-3',
+            'col-xs-offset-4',
+            'col-xs-offset-5',
+            'col-xs-offset-6',
+            'col-xs-offset-7',
+            'col-xs-offset-8',
+            'col-xs-offset-9',
+            'col-xs-offset-10',
+            'col-xs-offset-11',
+            'col-xs-offset-12',
+            'col-xs-pull-0',
+            'col-xs-pull-1',
+            'col-xs-pull-2',
+            'col-xs-pull-3',
+            'col-xs-pull-4',
+            'col-xs-pull-5',
+            'col-xs-pull-6',
+            'col-xs-pull-7',
+            'col-xs-pull-8',
+            'col-xs-pull-9',
+            'col-xs-pull-10',
+            'col-xs-pull-11',
+            'col-xs-pull-12',
+            'col-xs-push-0',
+            'col-xs-push-1',
+            'col-xs-push-2',
+            'col-xs-push-3',
+            'col-xs-push-4',
+            'col-xs-push-5',
+            'col-xs-push-6',
+            'col-xs-push-7',
+            'col-xs-push-8',
+            'col-xs-push-9',
+            'col-xs-push-10',
+            'col-xs-push-11',
+            'col-xs-push-12',
+            'control-label',
+            'danger',
```

Comment:
Won't this complain for any danger class? Similarly the rest of the generic named classes.

Reply:
Yes it would. I removed some of the very generic sounding class names (I commented them out as you can see) but not all.

Reply:
Better be more specific with the classes then. This can result in many false positives otherwise...

Reply:
Hm… I could change this in a list of selectors and use this instead. So in the case of `.danger` it would be `tr.danger, td.danger`. Would be some work.

Reply:
Can't you use a regex? Because it's very important to be as strict as possible with these to reduce false warnings.

Reply:
Please see the latest commit.